lkp-exec/install

```diff
@@ -282,6 +282,28 @@ install_base_support()
 {
 	install_packages "lkp" "$distro"
 
+	if [ "$distro" = "amazon_linux" -a "$_system_version" = "2" -a "$(gem --version)" != "3.2.3" ]; then
```

Comment:
thanks, could you mind wrap this in a function for now? Also help check the failure in "Ruby / test (2.5) (pull_request) Failing after 51s" which is related to shellcheck.